src/lib/updating-element.ts

```diff
@@ -304,7 +312,7 @@ export abstract class UpdatingElement extends HTMLElement {
       return;
     }
     const key = typeof name === 'symbol' ? Symbol() : `__${name}`;
```

Comment:
I could see a custom property definer wanting to customize this key. If you move this into the default property descriptor factory, then overriders can do whatever they want here.

Reply:
The key is for private property storage and is provided only so that users can follow the same convention as LitElement. Overriders can choose to use or ignore the key in the descriptor they return via the factory.

Reply:
This is now included in `createPropertyDescriptor(name, key, options)` so implementers can do whatever they want with it.